src/api/memberships.js

```diff
@@ -0,0 +1,37 @@
+import * as postgrest from './postgrest';
+
+export const postMembershipReq = membership => {
+  return postgrest
+    .postJSON('/memberships', membership, { Prefer: 'return=representation' })
+    .then(membershipJSON => membershipJSON[0])
+    .catch(err => console.log('Error posting membership:', err));
+};
+
+export const getMembershipsReq = filterOption => {
+  const filterKey = Object.keys(filterOption)[0];
+  const filterVal = filterOption[filterKey];
+  return postgrest
+    .getJSON(`/memberships?${filterKey}=eq.${filterVal}`)
+    .then(memberships =>
+      memberships.map(membership => {
+        return {
+          [membership.pursuance_id]: membership
+        };
+      })
```

Comment:
This prevents more than 1 person being a member of a pursuance, but let's not worry about this for today -- it's demo time :-D .

Reply:
(at least from Redux's perspective it does)

Reply:
Oh I was using that as a ref to the users memberships

Reply:
Not the memberships of the whole pursuance

Reply:
Soon lets make each key `${membership.pursuance_id}_${membership.user_username}` and the value an array

Reply:
Ooh I see. I haven't thought this through well enough other than to say that it seems like it will soon make sense to have the users listed in `AssignerSuggestions` populated based on `memberships`.

But we can table this.